src/main/java/Command/AddCommand.java

```diff
@@ -0,0 +1,27 @@
+package Command;
+
+import Ping.Task;
+import Ping.TaskList;
+import Ping.UI;
+import Ping.PingException;
+
+
+/**
+ * This class is used to add a task to the list of tasks
+ */
+public class AddCommand extends Command {
+    private Task task;
+    public AddCommand(Task task) {

```

Comment:
A linebreak here might make it more readable to distinguish between these two members
```suggestion
    private Task task;
    
    public AddCommand(Task task) {

```